jubakit/loader/postgresql.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ..base import BaseLoader
+from ..compat import *
+from psycopg2 import connect
+from psycopg2.extras import DictCursor
+from psycopg2 import sql
+
+class PostgreSQLoader(BaseLoader):
+  """
+  Loader to process columns of PostgreSQL.
+  """
+
+  def __init__(self, auth, **kwargs):
```

Comment:
The `table` must be required and I think the `table` should be an argument without keyword like below.
`def __init__(self, auth, table, **kwargs):`

Then the error handling code in L18-19 will not be necessary any more.